src/main/resources/standardTestCases/call_cache_capoeira/call_cache_capoeira_local.wdl

```diff
@@ -41,6 +41,15 @@ workflow call_cache_capoeira {
 
     # Different input names? Don't call cache!
     call read_files_inputs_renamed { input: ready = sleep.done, c = make_files.bananeira, d = make_files.balanca }
+    
+    # Like read_files but with 2 Array[File] instead of File
+    call read_array_files { input: ready = sleep.done, a = [make_files.bananeira], b = [make_files.balanca] }
+    
+    # Give the last call a chance to be writen to the call caching DB
+    call sleep as sleep_some_more { input: duration = 5, ready = read_array_files.done }
+    
+    # Same as read_array_files except a is empty and b contains both, in the same order. Don't call cache!
+    call read_array_files as read_array_files_rearranged { input: ready = sleep_some_more.done, a = [], b = [make_files.bananeira, make_files.balanca] }
```

Comment:
Yeah that would be nice, it would also add more testing to subworkflows. I'll make a ticket